YOLO-detect-buoys/README.md

```diff
@@ -1,14 +1,54 @@
 README.md
 
-This is the README file for the project by Vortex that uses YOLO to detect buoys.
+# YOLO detect buoys
 
-Usage:
+## This is the README file for the project by Vortex that uses YOLO to detect buoys.
+
+## Getting Started:
+
+### Prerequisites:
+
+Make sure that you have the following installed on your system:
+
+- [Python](https://www.python.org/)
+- [pip](https://pip.pypa.io/en/stable/installation/)
+- [venv](https://docs.python.org/3/library/venv.html)
+
+### Run the application:
 
 - Download the project dependencies by running `pip install -r requirements.txt`.
 - Create a virtual environment by running `python -m venv venv`.
```

Comment:
I might be wrong, but shouldn't you create the venv before you add deps to it?

Reply:
True, i mixed up